resolwe_bio/processes/reads_processing/trimmomatic.yml

```diff
@@ -9,55 +9,69 @@
   name: Trimmomatic (single-end)
   requirements:
     expression-engine: jinja
+    resources:
+      cores: 4
```

Comment:
Please run some performance tests and post the results to determine the best number of cores. If the performance scales linearly with the number of cores, we want to use more than 4 (see [this comment](https://github.com/genialis/resolwe-bio/pull/494#issuecomment-368896069)).